x-pack/plugins/observability_solution/apm/public/components/alerting/ui_components/alert_details_app_section/latency_alerts_history_chart.tsx

```diff
@@ -91,7 +95,7 @@ export function LatencyAlertsHistoryChart({
                 start,
                 end,
                 transactionType,
-                transactionName: undefined,
```

Comment:
@fkanout @benakansara Any idea why `transactionName` was undefined before?

Reply:
IIRC the transactionName was not added as filter and group by in APM latency rule when we implemented alert details page. So we didn't have transactionName information in the alert at the time.